utils/mk-squash-iso.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+help() {
+	cat <<EOF
+mk-squash-iso.sh [options]
+	Turns an sabotage rootfs into an bootable iso.
+	This requires an working isolinux installation on <rootfs>/boot.
+
+	-o <file>
+		Set the output file. Existing files will be overwritten
+		without asking. The file will be an iso image.
+
+	-l <label>
+		Define a custom disk label.
+
+	-r <dir>
+		Set the source rootfs. The rootfs itself will not be changed.
+
+	-t
+		Exclude tarballs directory.
```

Comment:
Usually, the /src/tarballs directory is empty anyways if you build from an chroot. This is due the bind mount in enter-chroot. This option just toggles if the tarballs folder as whole is ignored by mksquashfs or not. Not sure of putting the selection mechanism here would be appropiate.